donate/templates/payment/thank_you_master.html

```diff
@@ -22,14 +22,14 @@ <h1 class="heading heading--primary heading--bottom-margin">{% trans "Thank you
                     {% trans "I donated to @mozilla today because I #lovetheweb. Join me and help fight for a better and healthier internet, for all." as twitter_text context "Used as a tweet" %}
                     {% trans "I donated to Mozilla today" as email_subject context "Email subject line" %}
 
-                    <a class="button button--outline button--icon js-ga-track-click" href="https://www.facebook.com/sharer/sharer.php?u={{ request.scheme }}://{{ request.get_host }}/{% get_locale %}" title="{% trans 'Share on Facebook' %}" data-ga-category="Social" data-ga-action="Clicked on Button" data-ga-label="Facebook">
+                    <a class="button button--outline button--icon js-ga-track-click" href="https://www.facebook.com/sharer/sharer.php?u=https://foundation.mozilla.org" title="{% trans 'Share on Facebook' %}" data-ga-category="Social" data-ga-action="Clicked on Button" data-ga-label="Facebook">
```

Comment:
Should the links to `https://donate.mozilla.org/{% get_locale %}/thunderbird/` be updated in the TB template as well?

Reply:
Yeah, but I'm not sure to where quite yet/I'm not concerned about those right now. #597 has been filed.